mango/clients/aemet.py

```diff
@@ -0,0 +1,598 @@
+import logging
+import os
+from datetime import datetime
+from math import radians, cos, sin, asin, sqrt
+from typing import Union, Any, Tuple
+
+from tqdm import tqdm
+
+from mango.clients.rest_client import RestClient
+from mango.shared import ApiKeyError
+from mango.validators.aemet import *
+
+
+class AemetClient(RestClient):
+    """
+    This class will handle the connection to the AEMET API. It will allow to get the meteorological data from the
+    meteorological stations in Spain. To initialize the class, an API key is needed. This key can be obtained in the
+    AEMET website: https://opendata.aemet.es/centrodedescargas/altaUsuario
+
+    :param api_key: API key to connect to the AEMET API
+    :param wait_time: Wait time between requests in seconds. Default is 1.25 seconds
+    :doc-author: baobab soluciones
+
+    Usage
+    -----
+
+    >>> import load_dotenv
+    >>> import os
+    >>> from mango.clients.aemet import AemetClient
+    >>> load_dotenv()
+    >>> client = AemetClient(os.environ["AEMET_API_KEY"])
```

Comment:
What is it `load_dotenv`?

Reply:
It is a Python library to load .env files into enviroment variables. I agree It doesn't make sense in this doc example, We can assume user loaded AEMET_API_KEY into enviroment somehow